### PR TITLE
PATH help for Mac added

### PR DESCRIPTION
**Pull request by twhitt14**
I've seen lots of people search Stack Overflow for help on the PATH part of the quickstart (it was totally left out), so I thought I'd at least help out the Mac users - I'm not too familiar with Windows but I'm sure someone could whip up something similar to add for Windows as well.

### CODE REVIEW

File: _posts/documentation/learn/2000-01-01-quick-start.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ permalink: quick-start.html
 
 This instruction assumes that PhantomJS is installed and its executable is placed somewhere in the PATH.
 
+Mac PATH suggested setup:
+  1. Open Terminal.
+  2. Type `vi ~/.bash_profile` and hit enter (this opens or creates your bash_profile where you can customize Terminal commands).
+  3. Press `i` to enter insert/edit mode then type `alias phantomjs='~/PATH/TO/phantomjs'` and be sure to replace `~/PATH/TO/phantomjs` with something like `~/Documents/PhantomJS/bin/phantomsj` or wherever the file exists on your machine.
```
Comment (rajendraarora16) [inline _posts/documentation/learn/2000-01-01-quick-start.md]: In your ~/Documents/PhantomJS/bin/phantomsj? I think it should be phantomjs. Correct me if I am wrong?

Comment (applecrazy) [reply _posts/documentation/learn/2000-01-01-quick-start.md]: Better command, also is a one-liner: `echo "export PATH=$PATH:/location/of/phantomjs" >> .bash_profile`

Comment (applecrazy) [inline _posts/documentation/learn/2000-01-01-quick-start.md]: Better command, also is a one-liner: `echo "export PATH=$PATH:/location/of/phantomjs" >> .bash_profile`

Comment (twhitt14) [reply _posts/documentation/learn/2000-01-01-quick-start.md]: Oops yes that was a typo. should be `phantomjs`

**Pull request merged**